gerrypy/templates/map.jinja2

```diff
@@ -0,0 +1,163 @@
+{% extends "layout.jinja2" %}
+{% block title %}
+GerryPy
+{% endblock %}
+{% block nav_title %}<span>GerryPy</span>{% endblock %}
+{% block body %}
+    <div class='container'>
+        <form id='mapform'>
+{#             <label>State:</label>
```

Comment:
Normally I'd say that this is corpse code and should be deleted, *however* considering the way in which this app should expand it makes sense to leave this here.